src/services/opengin_service.py

```
@@ -8,91 +8,109 @@
-                return result    
-                
+                return result
+
         except NotFoundError:
-            raise       
+            raise
         except BadRequestError:
             raise
         except Exception as e:
-            logger.error(f'Read API Error: {str(e)}')
+            logger.error(f"Read API Error: {str(e)}")
```

Ruff (0.16.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)